README.md

```diff
@@ -268,6 +268,35 @@ keep it.
 Use [proofit404/blacken](https://github.com/proofit404/blacken).
 
 
+### PyCharm
+
+(Adapted from [*Pylinting with PyCharm*](https://kirankoduru.github.io/python/pylint-with-pycharm.html))
+
+1. Install `black`.
+
+        $ pip install black
+
+2. Locate your `black` installation folder.
+
+        $ which black         # MacOS/Linux
+        /usr/local/bin/black  # this is just a possible output check yours
+
+       <!-->
+
+        $ where black         # Windows
+        %LocalAppData%\Programs\Python\Python36-32\Scripts\black.exe  # possible location
+        
+3. Open External tools in PyCharm with `File -> Settings -> Tools -> External Tools`.
+
+4. Click the + icon to add a new external tool with the following values:
+    - Name: black
+    - Description: Black is the uncompromising Python code formatter.
+    - Program: <install_location_from_step_2>
+    - Arguments: $FilePath$
+
+5. Format the currently opened file by selecting `Tools -> External Tools -> black`.
```

Comment:
I would also add instructions on how to map it as a keymap. (You can simply do it through the regular keymap settings). It'll be there in External Tools

Reply:
I would simply link this guide: https://blog.jetbrains.com/webstorm/2016/08/using-external-tools/

Reply:
I have added the instructions as a bullet point on the next line.